content/guides/rpc/configure-solana-rpc-on-aws.md

```diff
@@ -0,0 +1,50 @@
+# Solana on AWS
```

Comment:
We use YAML frontmatter to handle the metadata displayed in the developer guides website.
I recommend the following YAML frontmatter:
```suggestion
---
date: Oct 26, 2023
difficulty: intermediate
featured: true
featuredPriority: 0
title: "Configure and run a Solana RPC on AWS"
description:
  "Running Solana nodes on AWS can help to reduce operational costs and improve reliability of your validators or RPC services"
tags:
  - rpc
  - infrastructure
keywords:
  - aws
  - amazon
  - node runner
---

# Solana on AWS
```